src/gui/torrentcontentfiltermodel.cpp

```diff
@@ -28,113 +28,122 @@
 {
-  return m_model->itemType(mapToSource(index));
+    return m_model->itemType(mapToSource(index));
 }
 
-int TorrentContentFilterModel::getFileIndex(const QModelIndex& index) const
+int TorrentContentFilterModel::getFileIndex(const QModelIndex &index) const
 {
-  return m_model->getFileIndex(mapToSource(index));
+    return m_model->getFileIndex(mapToSource(index));
 }
 
-QModelIndex TorrentContentFilterModel::parent(const QModelIndex& child) const
+QModelIndex TorrentContentFilterModel::parent(const QModelIndex &child) const
 {
-  if (!child.isValid()) return QModelIndex();
-  QModelIndex sourceParent = m_model->parent(mapToSource(child));
-  if (!sourceParent.isValid()) return QModelIndex();
-  return mapFromSource(sourceParent);
+    if (!child.isValid())
+        return QModelIndex();
+
+    QModelIndex sourceParent = m_model->parent(mapToSource(child));
+    if (!sourceParent.isValid())
+        return QModelIndex();
+
+    return mapFromSource(sourceParent);
 }
 
-bool TorrentContentFilterModel::filterAcceptsRow(int source_row, const QModelIndex& source_parent) const
+bool TorrentContentFilterModel::filterAcceptsRow(int sourceRow, const QModelIndex &sourceParent) const
 {
-  if (m_model->itemType(m_model->index(source_row, 0, source_parent)) == TorrentContentModelItem::FolderType) {
-    // accept folders if they have at least one filtered item
-    return hasFiltered(m_model->index(source_row, 0, source_parent));
-  }
-  return QSortFilterProxyModel::filterAcceptsRow(source_row, source_parent);
+    if (m_model->itemType(m_model->index(sourceRow, 0, sourceParent)) == TorrentContentModelItem::FolderType) {
+        // accept folders if they have at least one filtered item
+        return hasFiltered(m_model->index(sourceRow, 0, sourceParent));
+    }
+
+    return QSortFilterProxyModel::filterAcceptsRow(sourceRow, sourceParent);
 }
 
-bool TorrentContentFilterModel::lessThan(const QModelIndex &left, const QModelIndex &right) const {
-  switch (sortColumn()) {
-  case TorrentContentModelItem::COL_NAME: {
-    QString vL = left.data().toString();
-    QString vR = right.data().toString();
-    TorrentContentModelItem::ItemType leftType = m_model->itemType(m_model->index(left.row(), 0, left.parent()));
-    TorrentContentModelItem::ItemType rightType = m_model->itemType(m_model->index(right.row(), 0, right.parent()));
-
-    if (leftType == rightType)
-      return Utils::String::naturalCompareCaseInsensitive(vL, vR);
-    else if (leftType == TorrentContentModelItem::FolderType && sortOrder() == Qt::AscendingOrder)
-      return true;
-    else
-      return false;
-  }
-  };
-
-  return QSortFilterProxyModel::lessThan(left, right);
+bool TorrentContentFilterModel::lessThan(const QModelIndex &left, const QModelIndex &right) const
+{
+    switch (sortColumn()) {
+    case TorrentContentModelItem::COL_NAME: {
+        QString vL = left.data().toString();
```

Comment:
Wrong indentation.
Can be also fixed by extracting subroutine and removing curly braces.